<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>591</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Rejestracja</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>387</width>
     <height>540</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="registrationLabel">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zarejestruj się</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Imię</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="surnameLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Nazwisko</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="surnameEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="peselLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pesel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="peselLineEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="pinLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pinEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="emailLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="phoneNumberLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Numer telefonu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneNumberEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="dateOfBirthLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Data urodzenia</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateOfBirthEdit">
          <property name="toolTipDuration">
           <number>17</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="font">
           <font>
            <family>Bahnschrift SemiLight SemiConde</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Hasło</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="inputMask">
           <string/>
          </property>
          <property name="maxLength">
           <number>100</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="registerButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zarejestruj się</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Wyjdź</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
